src/main/java/uk/gov/pay/ledger/transaction/dao/mapper/TransactionMapper.java

```diff
@@ -45,17 +47,28 @@ public TransactionEntity map(ResultSet rs, StatementContext ctx) throws SQLExcep
                 .withMoto(rs.getBoolean("moto"))
                 .withGatewayTransactionId(rs.getString("gateway_transaction_id"))
                 .withGatewayPayoutId(rs.getString("gateway_payout_id"));
-        Source.from(rs.getString("source")).ifPresent(builder::withSource);
-        return builder.build();
+        Source.from(rs.getString("source")).ifPresent(transactionBuilder::withSource);
+        if (rs.getString("gateway_payout_id") != null) {
+            var payoutBuilder = aPayoutEntity()
+                    .withGatewayPayoutId(rs.getString("gateway_payout_id"))
+                    .withPaidOutDate(getZonedDateTime(rs, "paid_out_date").orElse(null));
+            transactionBuilder.withPayoutEntity(payoutBuilder.build());
+        }
+        return transactionBuilder.build();
     }
 
     private Long getLongWithNullCheck(ResultSet rs, String columnName) throws SQLException {
         long value = rs.getLong(columnName);
         return rs.wasNull() ? null : value;
     }
 
-    private Optional<ZonedDateTime> getZonedDateTime(ResultSet rs, String columnLabel) throws SQLException {
-        Timestamp timestamp = rs.getTimestamp(columnLabel);
+    private Optional<ZonedDateTime> getZonedDateTime(ResultSet rs, String columnLabel) {
+        Timestamp timestamp;
+        try {
+            timestamp = rs.getTimestamp(columnLabel);
+        } catch (SQLException ex) {
```

Comment:
I think we can just throw SQLException here as we will need to join payout all the time (search transactions as well as get one transaction).

Reply:
As discussed, it's a temporary solution until all queries have the payout table joined to transaction table.